src/Aspire.ProjectTemplates/templates/aspire-nunit/9.3/IntegrationTest1.cs

```diff
@@ -19,7 +19,7 @@ public class IntegrationTest1
     // public async Task GetWebResourceRootReturnsOkStatusCode()
     // {
     //     // Arrange
-    //     var cancellationToken = TestContext.CurrentContext.CancellationToken;
+    //     using var cancellationToken = TestContext.CurrentContext.CancellationToken;
```

Comment:
```suggestion
    //     var cancellationToken = TestContext.CurrentContext.CancellationToken;
```
Cancellation token isn't disposable.